backend/api/management/commands/seed.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from argparse import ArgumentParser
+
+from api.client import client
```

Comment:
Does it work with this import ?
I think it should either be `from ....api.client` (not really like this personally) or `from backend.api.client`

Reply:
Yeah it worked. `backend.api.client` is good too.

Reply:
Edit:

I checked again and now it's not working anymore. That's weird...